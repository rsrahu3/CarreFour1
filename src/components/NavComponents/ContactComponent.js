import * as React from 'react'
import Header from './../common/Header'
import TopHeader from './../common/TopHeader'

export default class ContactComponent extends React.Component {
  render () {
    
    console.log("contact component")
    return <div >
    <TopHeader/>
      <Header active="contact"/>
          This is Contact Component
    </div>
  }
}
