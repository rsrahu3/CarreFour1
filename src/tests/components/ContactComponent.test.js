import React from 'react';
import {shallow} from 'enzyme';
import ContactComponent from './../../components/NavComponents/ContactComponent';
import Adapter from 'enzyme-adapter-react-16';
import config from './configuartion';
import toJson from 'enzyme-to-json';



describe("ContactComponent Screen test",()=>{
    it('form check',()=>{
        const enzymeWrapper = shallow(<ContactComponent/>);
        expect(enzymeWrapper.find('div').text().includes('This is Contact Component')).toBe(true);
        expect(toJson(enzymeWrapper)).toMatchSnapshot();
    })
    
})