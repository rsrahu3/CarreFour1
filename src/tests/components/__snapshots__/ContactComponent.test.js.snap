// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactComponent Screen test form check 1`] = `
<div>
  <topHeader />
  <Header
    active="contact"
  />
  This is Contact Component
</div>
`;
